### ISSUE TITLE
Cannot reproduce the results of the paper

### ISSUE THREAD

**Issue by eeehco** [MengyuanChen21/ECCV2022-DELU#5]
Hi，authors.Thanks for your wonderful work！
I carried out the experiment on thmos14 according to the code you provided. The machine is nvidia GeForce 3090. I tried random seeds 0-4, but the highest one is still lower than your accuracy in the paper. Can you give me some suggestions.The average precision of random seeds from 0 to 4 is 45.7, 45.6, 45.4, 45.6, 46,1 respectively.
<img width="596" alt="Snipaste_2023-02-23_16-11-46" src="https://user-images.githubusercontent.com/102967148/220853143-bfb603ef-eab6-4fdd-93cf-be2850ce11a6.png">

**Comment by MengyuanChen21** [comment]
Thanks for your attention! As I answered in other issues, I ran the codes with random seeds 0 to 4, and obtained the results of 46.41, 46.30, 45.09, 45.84, and 45.88 (mAP 0.1-0.7), whose average is 45.9. The results you reproduced (45.7 on average) are only slightly lower than mine, which I think is within the normal range of error. The result 46.4 reported on the paper can be obtained by loading the checkpoint file I provided.
